articles/azure-large-instances/what-is-azure-large-instances.md

```diff
@@ -0,0 +1,197 @@
+Therefore, for ALI, Azure offers a consultation with a CSA/GBB in the field who can work with you.
+
+By the time your ALI implementation has been provisioned, the OS, networks, storage volumes, placements in zones and regions, and WAN connections between locations have already been configured.
+You're set to register your OS licenses (BYOL), configure the OS, and install the application layer.
+
+You'll see all the Azure Large instance resources, and their state and attributes, in the Azure portal.
+You can also operate the instances and open service requests and support tickets from there.
+
+## ALI stamp
+
+The Azure Large instance stamp itself combines the following components:
+
+* **Computing**
+Servers based on the generation of Intel Xeon processors that provide the necessary computing capability and are certified for the specialized workload.
+
+* **Network**
+A unified high-speed network fabric interconnects computing, storage, and LAN components.
+
+* **Storage**
+An infrastructure accessed through a unified network fabric.
+
+Within the multi-tenant infrastructure of the Azure Large instance stamp, customers are deployed in isolated tenants.
+When deploying a tenant, you name an Azure subscription within your Azure enrollment. 
+This Azure subscription is the one billed for your implementation of ALI.
+
+> [!Note]
+>A customer implementing ALI is isolated into a tenant.
+A tenant is isolated in the networking, storage, and compute layer from other tenants.
+Storage and compute units assigned to different tenants cannot see each other or communicate with each other on their implementations of ALI.
+
+## Operating system
+
+During the provisioning of the ALI, you can select the OS you want to install on the machines.
+
+>[!Note}
+> Remember, ALI is a BYOL model.
+
+The available Linux OS version is Red Hat Enterprise Linux (RHEL)8.4.  
+
+Microsoft loads base image with RHEL 8.4, but customers can choose to upgrade to newer versions in collaboration with Microsoft team.
+
+## Storage
+
+ALI provides highly redundant Fiber Channel storage.
+The infrastructure offers deep integration for enterprise workloads like SAP, SQL, and others.
+It also provides application-consistent data protection and data-management capabilities.
+The self-service management tools offer space-efficient snapshot, cloning, and granular replication capabilities along with single pane of glass monitoring. 
+The infrastructure enables zero Recovery Point Objective (RPO) and Recovery Time Objective (RTO) capabilities for data availability and business continuity needs.
+
+The storage infrastructure offers: 
+
+* Up to 4 x 100-GB uplinks.
+* Up to 32-GB Fiber channel uplinks.
+* All flash SSD and NVMe drive.
+* Ultra-low latency and high throughput.
+* Scales up to 4 PB of raw storage.
+* Up to 11 million IOPS.
+
+These Data access protocols are supported:
+
+* iSCSI
+* NVMe over FC
```

Comment:
Change this to Fiber Channel Protocol (FCP)

Reply:
Fixed 
To confirm in build